docs/extensions/overview.mdx

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 1
+sidebar_label: "Overview"
+id: extensions-overview
+title: Extensions
+description: Overview of Kwil's extensions system
+slug: /extensions/overview
+---
+
+Extensions are designed to allow developers to customize their Kwil network to fit their specific use case. Within the main Kwil implementation,
+there are exposed interfaces that can be implemented to customize the behavior of the node. Extensions are written in Go, and are compiled
+into the Kwil binary.
+
+Developers must take care to ensure that extensions are implemented properly. There are certain guidelines and restrictions that must be followed
+when implementing an extension. These are outlined in each extension types page. Failure to do so may result in node instability, consensus issues, or other problems.
+
+## Types of Extensions
```

Comment:
I think the sidebar should match the each type of extension name (with Oracles being a top level section with two subsections.)

For example:

This page would look like:

```
Types of Extensions
|- Authenticators
|- Kuneiform Precompiles
|- Oracles
    |- Resolutions
    |- Event Listener
```

And then the sidebar will have the same layout:

```
Extension
|- Overview
|- Building kwild With Extensions
|- Authenticators
    |- Authenticators Overview
    |- ... other current pages
|- Kuneiform Precompiles
    |- Precompiles Overview
    |- ... other current pages
|- Oracles
    |- Event Listeners
    |- Resolutions
```

This way, the sidebar aligns with how the content is presented in the overview. Thoughts?